scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hvssop.yaml

```diff
@@ -0,0 +1,55 @@
+FileHeader:
+  library_Suffix: 'SO'
+  device_type: 'HVSSOP'
+
+HVSSOP-10-1EP_3x3mm_P0.5mm_EP1.57x1.88mm:
+  size_source: 'https://www.ti.com/lit/ds/symlink/bq24090.pdf'
+
+  body_size_x:
+    nominal: 3
+    tolerance: 0.100
+  body_size_y:
+    nominal: 3
+    tolerance: 0.100
+  overall_height:
+    maximum: 1.1
+
+  overall_size_x:
+    nominal: 4.9
+    tolerance: 0.152
+  lead_width:
+    minimum: 0.17
+    maximum: 0.27
+  lead_len:
+    nominal: 0.53
+    tolerance: 0.152
+
+  pitch: 0.5
+  num_pins_x: 0
+  num_pins_y: 5
+
+  EP_size_x:
+    nominal: 1.57
+    tolerance: 0
+  # EP_size_x_overwrite:
+  # EP_mask_x:
+  EP_size_y:
+    nominal: 1.88
+    tolerance: 0
+  # EP_size_y_overwrite:
+  # EP_mask_y:
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [2, 2]
+
+  thermal_vias:
+    count: [2, 2]
+    drill: 0.2
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
+    EP_num_paste_pads: [2, 2]
+    # paste_between_vias: 1
+    # paste_rings_outside: 1
+    EP_paste_coverage: 0.7
+    grid: [1.1, 1.3]
```

Comment:
Can you elaborate why you choose any of those value?
Why 2x2 paste pads? Why the 0.7 coverage, why selecting that grid?

Reply:
Well, same as the other... The MSOP-10 did work well on the bq24090, so I used this one.

- I don't know if it is possible to have a fifth thermal vias, at the centre

Reply:
I looked at the generator and could not find a feature for the 5th center thermal via.
But the 4 generated vias, when using the default settings are fine. They are according to IPC recommendations.

Reply:
ok great !

Reply:
You mean, should I remove all param except count ?

Reply:
Removed all non necessary params